docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -109,6 +112,29 @@ When a Global Admin invites a user to join an account, the Global Admin selects
   - Has Namespace Admin permissions for each Namespace created by that user
 - **Read-Only:** Can only read information
 
+:::note Default Role
+
+When the account is created, the initial user who logs in is automatically assigned the Account Owner role.
+If your account does not have an Account Owner, please reach out to [Support](https://temporalsupport.zendesk.com/) to assign the appropriate individual to this role.
+
+:::
+
+## Using the Account Owner Role
+
+The Account Owner role (i.e., users with the Account Owner system role) holds the highest level of access in the system.
+This role configures account-level parameters and manages Temporal billing and credit data.
+It allows users to perform all actions within the Temporal Cloud account.
+
+:::tip Best Practices
+
+Temporal strongly recommends the following precautions when assigning the AccountOwner role to users:
+
+- Assign the role to at least two users in your organization.
+  Otherwise, limit the number of users with this role.
+- Associate each Account Owner’s email address with their role so Temporal Cloud support can contact them in urgent situations.
```

Comment:
question: what does this mean? To associate the account owners email address with their role?

Reply:
I think that's a team question, rather than a docs question

Reply:
@UPatel12 Do you know what this is referring to?

Reply:
I'm going to do a rephrase on this to:

"Associate an email address with each Account Owner so Temporal Cloud support can be in contact in urgent situations"

Reply:
Marking this resolved so I don't have unresolved feedback under my next push.

Reply:
intention is to tell the users "It's best practice to assign this role to a real user's email instead of a service or team email"